cookiecutter.json

```diff
@@ -2,6 +2,6 @@
   "package_name": "package.name",
   "namespace": "{{ cookiecutter.package_name.split('.')[0] }}",
   "name": "{{ cookiecutter.package_name.split('.')[1] }}",
-  "project_slug": "{{ cookiecutter.package_name.lower().replace('.', '') }}",
+  "project_slug": "{{ cookiecutter.name.lower() }}",
```

Comment:
looks a lot better to me.. let me know if I keep like this or change

Reply:
It is unclear to me from the name what this is.

Reply:
it is in the line right above, just taking off the module name from the slug, instead of ``collective.faq`` slug would be just ``faq``

Reply:
I liked the way it was before, because this generates (for a `package.name` package):

```
layer="package.name.interfaces.INameCoreLayer"
```

For example, in Plone we always generate the whole name like:
https://github.com/plone/plone.app.contenttypes/blob/master/plone/app/contenttypes/interfaces.py#L5

```
class IPloneAppContenttypesLayer(Interface):
```

Reply:
I will change it and merge.